kinto_http/utils.py

```diff
@@ -44,3 +47,13 @@ def chunks(l, n):
             yield l[i:i+n]
     else:
         yield l
+
+
+def json_serial(obj):
+    """JSON serializer for objects not serializable by default json code"""
+    if isinstance(obj, (datetime, date)):
+        return obj.isoformat()
+    raise TypeError("Type %s is not serializable" % type(obj))
```

Comment:
`return json.JSONEncoder.default(obj)`?

Reply:
If I understood correctly this is already a default in case the default one failed to encode.